File: src/main/java/com/codethen/api/StatusController.java
package com.codethen.api;

import jakarta.ws.rs.Produces;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class StatusController {

    /** Simple message, just to know that the new version is up */
    @GetMapping("status")
    @Produces("plain/text")
    public String status() {
        return "v0.7.0 - Translation via Google";
    }
}

File: src/main/kotlin/com/codethen/api/StatusController.kt
package com.codethen.api

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class StatusController {

    /** Simple message, just to know that the new version is up */
    @GetMapping("status")
    fun status() : String {
        return "v0.6.2 - Throttle translations, save searches (with date)"
    }
}